src/app/data-entries/select/select-form/select-form.component.html

```diff
@@ -0,0 +1,15 @@
+<form>
+    <div class="select-wrapper">
+        <igx-select #selectModel="ngModel" [(ngModel)]="selected" [ngModelOptions]="{standalone: true}" required>
```

Comment:
Wouldn't mind this using a name inside the form TBH:
```suggestion
        <igx-select #selectModel="ngModel" [(ngModel)]="selected" name="fruit" required>
```